requirements/base.txt

```diff
@@ -40,7 +40,7 @@ importlib-metadata==4.10.1
     #   pytest
 iniconfig==1.1.1
     # via pytest
-multidict==5.2.0
+multidict==6.0.2
```

Comment:
Dropped support for `Python 3.6`. One major change in `6.0.0` was reverted in `6.2.0` back to same as version `5.2.0`. See changes on https://github.com/aio-libs/multidict/blob/master/CHANGES.rst#602-2022-01-24.